### ISSUE TITLE
Deletion across multiple namespaces

### ISSUE THREAD

**Issue by bm1216** [AvitalTamir/cyphernetes#83]
### Problem
The tool doesn't seem to support deletion across namespaces(?) when in `ALL NAMESPACES` mode.
```
(MY_CLUSTER) ALL NAMESPACES » MATCH (p:Pod) 
WHERE p.status.containerStatuses[0].state.terminated.reason="Completed" 
DELETE p;

Query executed in 1.034922794s
```

The above doesn't do anything even though it does MATCH atleast a few resources. 
Let me know if i'm doing anything wrong please.

**Comment by AvitalTamir** [comment]
Hey thanks so much for reporting! The query looks legit, will look into it.

**Comment by AvitalTamir** [comment]
Resolved in v0.11.1
